packages/app/tasks/store-pool-rewards.js

```diff
@@ -0,0 +1,43 @@
+const { trace, info, error, arg } = require('../utils/logger');
+const DB = require('../utils/db');
+
+const BATCH_SIZE = 25;
+
+const storePoolRewardsTask = async (env, { poolToken }) => {
+    const storePoolRewards = async (data) => {
+        info('Storing pool rewards for', arg('poolToken', poolToken), arg('batchSize', BATCH_SIZE));
+
+        let totalGas = 0;
+
+        const providers = Object.keys(data.pendingRewards);
+        for (let i = 0; i < providers.length; i += BATCH_SIZE) {
+            const providersBatch = providers.slice(i, i + BATCH_SIZE);
+
+            for (let j = 0; j < providersBatch.length; ++j) {
```

Comment:
Minor: can use `for (const provider of providersBatch)`.

Reply:
👍